kitsune/sumo/jinja2/base.html

```diff
@@ -286,14 +286,16 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox for Developers') }}</h5>
           </ul>
         </section>
 
-        <section class="mzp-c-footer-col">
-          <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
-          <ul class="mzp-c-footer-list">
-            <li><a href="{{ url('users.auth') }}">{{ _('Sign In/Up') }}</a></li>
-            <li><a href="https://www.mozilla.org/en-US/firefox/accounts/">{{ _('Benefits') }}</a></li>
-            <li class="mzp-c-footer-heading"><br><a href="https://fpn.firefox.com/">{{ _('Firefox Private Network') }}</a></li>
-          </ul>
-        </section>
+        {% if not settings.READ_ONLY %}
```

Comment:
Perhaps we want this to just wrap the "Sign In/Up" link so we don't clobber the FPN link too?